src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -134,17 +132,33 @@ private void extractPublisher() {
      * @param dcSchema Metadata in DublinCore format.
      */
     private void extractBibTexFields() {
-        List<String> relationships = dcSchema.getRelations();
-        if (relationships != null) {
-            for (String r : relationships) {
-                if (r.startsWith("bibtex/")) {
-                    r = r.substring("bibtex/".length());
-                    int i = r.indexOf('/');
-                    if (i != -1) {
-                        bibEntry.setField(r.substring(0, i), r.substring(i + 1));
+        Predicate<String> isBibTeXElement = s -> s.startsWith("bibtex/");
+        Consumer<String> splitBibTeXElement = s -> {
+            // split solution is complicated, because some fields contains url etc.
+            String temp = s.substring("bibtex/".length());
+            int i = temp.indexOf('/');
+            if (i != -1) {
+                String key = temp.substring(0, i);
+                String value = temp.substring(i + 1);
+                bibEntry.setField(key, value);
+
+                // only for month field - override value
+                // workaround, because the date value of the xmp component of pdf box is corrupted
+                if ("month".equals(key)) {
```

Comment:
We are in the special bibtex mode. So, we hit `bibtex/month/#mar#` in this case, didn't we? Did we hit `bibtex/month/mar`? In the former case, I would assume, the export is corrupted. In the latter case, I don't know, what's happening. In all cases, it strange that the date value of PdfBox is corrupt; we are in our custom bibtex mode 😇